cumulusci/tasks/push/tasks.py

```diff
@@ -283,40 +287,43 @@ def _run_task(self):
                 "Scheduling push for %d minutes from now", delay_minutes
             )
             start_time = datetime.utcnow() + timedelta(minutes=delay_minutes)
+        if not self.options["dry_run"]:
+            self.logger.warning(
+                "Skipping push job creation due to dry_run flag enabled."
+            )
```

Comment:
This log line looks like it's in the wrong place -- it says dry_run is enabled in the case where it's not.

Also let's log at the info level, not warning, and be sure to output how many orgs are selected.